src/test/ui/parser/pat-tuple-1.rs

```diff
@@ -1,5 +1,6 @@
 fn main() {
-    match 0 {
-        (, ..) => {} //~ ERROR expected pattern, found `,`
+    match (0, 1) {
+        (, ..) => {}
+        //~^ ERROR expected pattern, found `,`
```

Comment:
Moving annotations to separate lines should no longer be necessary with test output filtering.
(I personally prefer trailing annotations because they are more compact.)